data/annotation/metabolites.tsv

```diff
@@ -999,7 +999,7 @@ mets	metsNoComp	metBiGGID	metKEGGID	metHMDBID	metChEBIID	metPubChemID	metLipidMa
 "m00602r"	"m00602"	""	""	""	""	"53481513"	""	"CE3554"	""	"CE3554"	"MNXM35505"	"m00602r"
 "m00603c"	"m00603"	""	"C05497"	""	""	""	""	""	""	"M00603"	"MNXM3472"	"m00603c"
 "m00604c"	"m00604"	""	"C13713"	"HMDB00879"	"CHEBI:805752"	"101771"	""	"CE5072"	""	"CE5072"	"MNXM8277"	"m00604c"
-"m00605c"	"m00605"	""	"C05485"	""	"CHEBI:28043"	""	"LMST02030167"	""	""	"M00605"	"MNXM4305"	"m00605c"
+"m00605c"	"m00605"	"21hprgnlone"	"C05485"	""	"CHEBI:28043"	""	"LMST02030167"	""	""	"M00605;21hprgnlone_c"	"MNXM735991"	"m00605c"
```

Comment:
This and the other ID are present in the Recon3D **model** (and "reconstruction"). The problem is that on VMH, they strip off the compartment abbreviation (`_c`). The original aim was to use these `Recon3D` fields to help maintain compatibility with the Recon3D GEM. However, since these Recon3D IDs are supposed to link to VMH, maybe we should consider stripping off the compartment abbreviation.

Reply:
I see. How probable is the scenario of one needing to map to Recon3D specifically, and not VMH? If the Recon3D ids are needed, I would suggest renaming the current column to VMH and adding a new one, that is not supposed to have a valid link.

Reply:
I think it's probably better to remove the compartment abbreviations and have them properly link to VMH. If someone is interested in re-mapping to the Recon3D model, they could simply append the compartment abbreviation to all the "Recon3D" (VMH) IDs.